### ISSUE TITLE
Remove a11y_text and excessive tabindexes

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2075]
### Subject of the issue
Issue reported of 'gratuitous use of tabindex' by Facebook, Barclays and S.A.P.
Reference [understanding screen reader interaction modes](https://tink.uk/understanding-screen-reader-interaction-modes/)

### Steps to reproduce
Turn on accessibility mode.
Tab.

### Expected behaviour
Only form elements and links should be tabbable.

### Actual behaviour
All text is tabbable.

### Resolves
#2040 li in accessibility mode requires two tab presses
#2082 links don't open properly in accessibility mode

**Comment by oliverfoster** [comment]
works completed
